wagtail/images/models.py

```diff
@@ -376,6 +376,28 @@ def usage_url(self):
     def __str__(self):
         return self.title
 
+    def __eq__(self, other):
```

Comment:
This custom equality method is needed so that image instances with the same ID but different alt text / is_decorative flags do not compare as equal, which would cause the block to be omitted from the comparison view. It's largely copied from [Django's `Model` implementation](https://github.com/django/django/blob/99dcc59237f384d7ade98acfd1cae8d90e6d60ab/django/db/models/base.py#L593) - my attempts to shorten this with `super()` failed (for some reason `super().__eq__(None)` returns true).

Reply:
Docstring added in a7d99dadbef7015ec5f1cf251072c32ddc24c581.